### PR TITLE
Use compiled executable in tests

### PR DESCRIPTION
**Pull request by dokshina**
Compile cartridge executable and use it as a fixture in all tests

### CODE REVIEW

File: test/python/project.py [hunk 1]
```diff
@@ -51,13 +51,13 @@ def is_removed_on_pack(filename):
 # Class Project
 ###############
 class Project:
-    def __init__(self, name, basepath, template='cartridge'):
+    def __init__(self, cartridge_cmd, name, basepath, template='cartridge'):
```
Comment (olegrok) [inline test/python/project.py]: Could we eliminate passing this fixture in each test if you've already used it in "Project" class? Seems we could export it as Project.cmd or smth similar.

Comment (dokshina) [reply test/python/project.py]: I'm not sure that `cartridge_cmd` is a part of Project.

Passing this fixture in each tests is clear and doesn't confuse.

Comment (olegrok) [reply test/python/project.py]: IMO If you have a "Project" you should have a way to run it

Comment (dokshina) [reply test/python/project.py]: Yes, but `cartridge_cmd` doesn't belong to a project.

Comment (olegrok) [reply test/python/project.py]: But you've already used it to build. I don't see reasons why it should stop us from using to run.

Comment (dokshina) [reply test/python/project.py]: I sure that `project.cartridge_cmd` will confuse.
And I don't see any problems in passing this fixture, this command is the main object of testing, why shouldn't we pass it explicitly?

Comment (olegrok) [reply test/python/project.py]: ok

File: test/helper.lua [hunk 2]
```diff
@@ -1,5 +1,6 @@
 jit.off()
```
Comment (olegrok) [inline test/helper.lua]: wow

Why?

**Pull request merged**